qutip/bloch.py

```diff
@@ -751,17 +771,17 @@ def plot_points(self):
                 self.axes.scatter(real(self.points[k][1][indperm]),
                                   -real(self.points[k][0][indperm]),
                                   real(self.points[k][2][indperm]),
-                                  s=s, alpha=1, edgecolor=None,
-                                  zdir='z', color=pnt_colors,
-                                  marker=marker)
+                                  s=s, alpha=self.point_alpha[k],
+                                  edgecolor=None, zdir='z',
+                                  color=pnt_colors, marker=marker)
 
             elif self.point_style[k] == 'l':
                 color = self.point_color[mod(k, len(self.point_color))]
                 self.axes.plot(real(self.points[k][1]),
                                -real(self.points[k][0]),
                                real(self.points[k][2]),
-                               alpha=0.75, zdir='z',
-                               color=color)
+                               alpha=self.point_alpha[k],
```

Comment:
Here we are changing the default alpha from 0.75 to 1.0. I guess we could either have the default in add points depend on the point style used (seems a bit complicated) or add a note to the docstring for `add_points` stating that the default for points with lines changed from 0.75 to 1.0. I think I favour documenting the change, rather than making the implementation more complex.